aquadoggo/Cargo.toml

```diff
@@ -19,7 +19,6 @@ edition = "2018"
 anyhow = "^1.0.62"
 async-graphql = { version = "^5.0.6", features = ["dynamic-schema"] }
 async-graphql-axum = "^5.0.6"
-async-recursion = "^1.0.0"
```

Comment:
I discovered two unused dependencies using `cargo udeps`. Tests are passing so I'm guessing they are indeed not needed but please double-check.